Docs/ros_launchs.md

```diff
@@ -46,7 +45,7 @@ Spawns an ego vehicle with sensors attached and waits for world information.
 
 ---
 ## carla_example_ego_vehicle.launch
-Based on [carla_ego_vehicle.launch](#carla-ego-vehicle-launch), spawns an ego vehicle (`role-name="ego_vehicle"`) using a provided file describing the sensors. Said file can be found in: `share/carla_ego_vehicle/config/sensors.json`. 
+Based on [carla_ego_vehicle.launch](#carla-ego-vehicle-launch), spawns an ego vehicle (`role-name="ego_vehicle"`) using a provided file describing the sensors. Said file can be found in `/share/carla_ego_vehicle/config/sensors.json`. 
```

Comment:
At "Said file can be found in `/share/carla_ego_vehicle/config/sensors.json`."
The path should be something like

If you installed ROS-Bridge using APT repository '/opt/carla-ros-bridge/melodic/share/carla_ego_vehicle/config/sensors.json'

or If you installed using source repository '<carla-ros-bridge PATH>/catkin_ws/src/ros-bridge/carla_ego_vehicle/config/sensors.json'

Reply:
Updated!